dnd5e/dnd5e.d.ts

```diff
@@ -124,18 +124,6 @@ declare namespace DND5e {
   type AreaTarget = 'cone' | 'cube' | 'cylinder' | 'line' | 'radius' | 'sphere' | 'square' | 'wall';
   type TargetType = AreaTarget | 'ally' | 'creature' | 'enemy' | 'none' | 'object' | 'self' | 'space';
 
-  type DurationType =
```

Comment:
So DND5e.DurationType is NOT the same as DND5e.TimePeriod. For reference this is DND5e.TimePeriod:
```ts
type TimePeriod = 'inst' | 'turn' | 'round' | 'minute' | 'hour' | 'day' | 'month' | 'year' | 'perm' | 'spec';
```
I was writing the types based upon the template.json etc and these abbreviations are used. Maybe I mixed them up? But this is definitely in the config. You might be pulling the translated versions where these are the keys and `instantaneous` etc. is the English translation, say because of PrepareData difference (see https://github.com/League-of-Foundry-Developers/foundry-vtt-dnd5e-types/issues/18)